scripts/release/orthoinference/deprecated/wrapper_ortho_inference.pl

```diff
@@ -0,0 +1,144 @@
+# change the source species, restrict the run to one target species, or indicate a source database other than the
+# default test_slice_reactomeversion_myisam. It's also possible to limit inference to specific Events by giving the
+# internal id of the upstream event(s) on the command line. Inference will then be performed for these Events and all
+# their downstream Events.
+
+use lib '/usr/local/gkb/modules';
+
+use GKB::Config;
+use GKB::DBAdaptor;
+use GKB::Instance;
+use GKB::Utils_esther;
+use GKB::Config_Species;
+
+use Data::Dumper;
+use DBI;
+use English qw/-no_match_vars/;
+use Getopt::Long;
+use List::MoreUtils qw/any/;
+use Readonly;
+
+use Log::Log4perl qw/get_logger/;
+Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+@ARGV || die "Usage: $PROGRAM_NAME -source_db [source database] -target_db [database_to_create] " .
+    '-source_species [species_to_infer_from - default hsap] -target_species [species_to_infer_to - default all] ' .
+    '-release [Reactome version number] -release_date [scheduled release date as yyyy-mm-dd]';
+
+my ($user, $pass, $host, $port, $source_db, $target_db, $source_species, $target_species, $release, $release_date, $debug);
+GetOptions(
+    'user:s' => \$user,
+    'pass:s' => \$pass,
+    'host:s' => \$host,
+    'port:i' => \$port,
+    'source_db:s' => \$source_db,
+    'target_db:s' => \$target_db,
+    'source_species:s' => \$source_species,
+    'target_species:s' => \$target_species,
+    'release:i' => \$release,
+    'release_date:s' => \$release_date,
+    'debug' => \$debug
+);
+$source_db || die "Need source database: -source_db [database from which to copy content]\n";
+$release || die "Need release version: -release [Reactome version number]\n";
+$release_date || die "Need release date: -release_date [scheduled release date as yyyy-mm-dd]\n";
+$user ||= $GKB::Config::GK_DB_USER;
+$pass ||= $GKB::Config::GK_DB_PASS;
+$host ||= $GKB::Config::GK_DB_HOST;
+$port ||= $GKB::Config::GK_DB_PORT;
+$target_db ||= 'release_current';
+$source_species ||= 'hsap'; # Human is defined as default source species
+
+my %db_options = (
+    '-host' => $host,
+    '-port' => $port,
+    '-user' => $user,
+    '-pass' => $pass,
+);
+my $db_option_string = create_db_option_string(\%db_options);
+
+#create database handle - needed to create the source database copy which will be used to add orthology data
+my $dbc = DBI->connect(
+    "DBI:mysql:database=$source_db;host=$host;port=$port",
+    $user,
+    $pass,
+    { RaiseError => 1, AutoCommit => 1}
+);
+if (!$dbc) {
+    $logger->error_die("Error connecting to database; $DBI::errstr\n");
+}
+
+# Create and populate target db from source db
+system("mysql -u$user -p$pass -e 'drop database if exists $target_db; create database $target_db'") == 0 or die "$?";
+run("mysqldump --opt -u$user -p$pass -h$host $source_db | mysql -u$user -p$pass -h$host $target_db") == 0 or die "$?";

```

Comment:
Doesnt `create_orthoinference_db.pl` do the same thing?

Reply:
Yes, sorry I should have mentioned that this script is in the deprecated folder now.  It just showed up as new because I made some edits before moving it.

Reply:
A brand-new script that's already deprecated, before merging back?